### ISSUE TITLE
[BUG] Floating point height with initialTopMostIndex does not render on certain scale

### ISSUE THREAD

**Issue by Elabar** [petyosi/react-virtuoso#463]
**Describe the bug**
When the height of the container is floating point number, the list seems to be not rendering when initialTopMostIndex is specified on certain scale.

**Reproduction**
Code: https://codesandbox.io/s/upbeat-mccarthy-2zwqq?file=/src/App.js
Preview link: https://2zwqq.csb.app/

**To Reproduce**
1. Zoom to different scale
2. Refresh page
3. Observe the result

**Expected behavior**
List should be rendered properly.

**Screenshots**
![image](https://user-images.githubusercontent.com/15872787/136146119-e34ff2be-785e-4960-bc03-4dd9f06465ba.png)

**Additional context**
On windows laptop, most of the display setting is 1920x1080 with 125% scaling (default). The 125% scaling makes the list not rendering properly even when the browser scaling is 100%.
Also, this seems to be a problem with Chromium base browser only? Can't really reproduce this issue in Firefox.

For now, I am using [AutoSizer](https://github.com/bvaughn/react-virtualized-auto-sizer) to mitigate this issue.

**Comment by petyosi** [comment]
Thanks, I will take a look.

**Comment by Elabar** [comment]
Update: Autosizer wont actually fix this issue, the height of the container might still be a floating point number after some scaling despite setting the height as an integer explicitly.

For example: 
![image](https://user-images.githubusercontent.com/15872787/136200486-7a7cc309-02f2-467f-8901-390421d2c2f0.png)

**Comment by petyosi** [comment]
Yeah, I did some back and forth on floating point numbers. Will look into fixing this by the end of this week
